### PR TITLE
Fix: Logo and Hamburger overlap mobile + markup hour

### PR DESCRIPTION
**Pull request by thet**
Ref: scrum-953

In surveys in mobile view, there is now a tree toggler:

image

Expanded view:

image

This PR‌ contains this change plus some simplifications and bug fixes.

Please see the individual commits.

### CODE REVIEW

**Review by ale-rt — APPROVED** [review]
LGTM

File: src/euphorie/client/browser/webhelpers.py [hunk 1]
```diff
@@ -1170,5 +1170,146 @@ def is_new_session(self):
...
+                "href": f"{url}/@@start#content",
+                "title": api.portal.translate(
+                    _("label_preparation", default="Preparation")
+                ),
+                "has_tree": False,
+            }
+        )
+
+        if self.use_involve_phase:
+            active = section == "involve"
```
Comment (reinhardt) [inline src/euphorie/client/browser/webhelpers.py:1205]: Reusing the `active` and `disabled` variables might lead to bugs when setting them is omitted by accident.

**Review by reinhardt — APPROVED** [review]
Just one small thing that I'll leave to your discretion.

File: src/euphorie/client/browser/webhelpers.py (hunk 1 above)
Comment (thet) [reply src/euphorie/client/browser/webhelpers.py:1205]: tnx for the review, but I‌ think I'll keep it as it is currently.
Let's do it in another step.
I'm happy for simplifications of this method anyways!

**Pull request merged**